spec/support/forgot_password_page.rb

```diff
@@ -0,0 +1,7 @@
+class ForgotPasswordPage < SitePrism::Page
+  set_url '/users/password/new'
+  set_url_matcher %r{/users/password/new}
+
+  element :email_field, '#user_email'
```

Comment:
I was under the impression that IDs or test classes are acceptable — but I've been a bit lax even in that regard. Will check opinions & fix